static/js/components/Dashboard.js

```diff
@@ -7,14 +7,45 @@ import { makeProfileImageUrl } from '../util/util';
 
 class Dashboard extends React.Component {
```

Comment:
Can you rename this to `Header` or `DashboardHeader`? It feels like that would be more descriptive of the role of this component.